kernel-install/50-mkosi.install

```diff
@@ -80,45 +59,17 @@ def main() -> None:
         allow_abbrev=False,
         usage="50-mkosi.install COMMAND KERNEL_VERSION ENTRY_DIR KERNEL_IMAGE INITRD…",
```

Comment:
Please move the entire argument parser into the `parse()` method as well and pass in the name and description of the plugin as arguments.

Reply:
why? it's just going to be more convoluted, as it's different across scripts anyway

Reply:
It's not, it's 100% identical for each kernel-install plugin, only the description and name are different, all the rest is identical because kernel-install determines the interface, not an individual plugin.